srcpkgs/boilr/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'boilr'
+pkgname=boilr
+version="0.3.0"
+revision=1
+build_style=go
+only_for_archs="x86_64"
```

Comment:
why ?

Reply:
Sorry, my mistake. Had newbie problems compiling and tried that as a workaround. Forgot to remove.